src/__tests__/settings/Settings.test.tsx

```diff
@@ -22,17 +22,16 @@ describe('Settings', () => {
     const history = createMemoryHistory()
     history.push('/settings')
 
-    const wrapper = mount(
+    // eslint-disable-next-line react/prop-types
+    const Wrapper: React.FC = ({ children }) => (
       <Provider store={store}>
         <Router history={history}>
-          <TitleProvider>
-            <Settings />
-          </TitleProvider>
+          <TitleProvider>{children}</TitleProvider>
         </Router>
-      </Provider>,
+      </Provider>
     )
 
-    return wrapper
+    return render(<Settings />, { wrapper: Wrapper })
```

Comment:
A little bit of refactor work on some code written a while back: 

Could we have this `setup` function return an object that has one property container where container is the value returned by the `render` function? This will make it consistent with other setup functions and the direction we are trying to move in.

Reply:
I know we have all had conversations in KCD Discord chat I just wanted to document here as well. Thanks for all your help Jack 😄 

React Testing Library already returns a `container` object as part of its `RenderResults` though you will notice its rarely used unlike Enzyme, that is by design since the `container` is generally utilized in inaccessible ways, like getting elements based on id's, classes, props etc... 

For the most part, the `screen` object from the library is utilized with the most accessible query possible to trigger user behaviors in a meaningful way and assert those behaviors changed, updated, or otherwise affected things.